docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-meraki-source.md

```diff
@@ -9,20 +9,14 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
   <meta name="robots" content="noindex" />
 </head>
```

Comment:
```suggestion

```

Reply:
@carlos-castillo-jask, remove these lines so that the article can be indexed for browser searches.

Reply:
Done